code/sheet_generator.py

```diff
@@ -24,6 +24,10 @@
     'GetZakelijkGerechtigden',
     'GetKadasterPersonen',
     'GetKadasterNietNatuurlijkPersonen',
+    'GetBeslagenKadastraalOnroerendeZaak',
+    'GetHypothekenKadastraalOnroerendeZaak',
+    'GetPrivaatrechtelijkeBeperkingen',
+    'GetPubliekrechtelijkeBeperkingen',
```

Comment:
PubliekrechtelijkeBeperkingen zijn ge-excluded omdat dit anders niet gaat werken? Is namelijk een andere vorm van endpoint...

Reply:
klopt, het script kan alleen enkele resources aan, geen collecties. die moet ik dus helaas nog met de hand doen